ui/page/root/home_page.go

```diff
@@ -230,6 +304,11 @@ func (hp *HomePage) initDEX() {
 					}),
 				)
 			}).
+			SetNegativeButtonText(values.String(values.StrLoginLater)).
+			NegativeButtonStyle(hp.Theme.Color.OrangeRipple, hp.Theme.Color.Surface).
```

Comment:
Use the same style for the "I'll sync later" button. This "Login later" button shouldn't look like a primary button, it should be less visible.